integrated_channels/api/v1/blackboard/serializers.py

```diff
@@ -18,13 +18,18 @@ class Meta:
         extra_fields = (
             'client_id',
             'client_secret',
+            'encrypted_client_id',
+            'encrypted_client_secret',
             'blackboard_base_url',
             'refresh_token',
             'uuid',
             'oauth_authorization_url',
         )
         fields = EnterpriseCustomerPluginConfigSerializer.Meta.fields + extra_fields
 
+    encrypted_client_id = serializers.CharField(required=False, allow_blank=False, read_only=False)
+    encrypted_client_secret = serializers.CharField(required=False, allow_blank=False, read_only=False)
+
```

Comment:
we can remove this part and do the replacing here and everywhere else in a followup PR

Reply:
@sameenfatima78 had to change the scope of this PR to avoid test coverage failure.